server/src/main/java/com/objectcomputing/checkins/services/skills/SkillController.java

```diff
@@ -0,0 +1,106 @@
+package com.objectcomputing.checkins.services.skills;
+
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.HttpStatus;
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.*;
+import io.micronaut.security.annotation.Secured;
+import io.micronaut.security.rules.SecurityRule;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Nullable;
+import javax.inject.Inject;
+import javax.validation.Valid;
+import java.net.URI;
+import java.util.List;
+import java.util.UUID;
+
+@Controller("/skill")
+@Secured(SecurityRule.IS_ANONYMOUS)
+@Produces(MediaType.APPLICATION_JSON)
+@Tag(name="skill")
+public class SkillController {
+
+    private static final Logger LOG = LoggerFactory.getLogger(SkillController.class);
+
+    @Inject
+    private SkillRepository skillRepo;
```

Comment:
It looks like Logger and skillRepo are unused in this class. Do we still need it?

Reply:
@Bagurp skillRepo is being used on line 35. 
@hwilliams314 can we just rename it to skillRepository and i learned that we don't need @Inject annotation. Micronaut figures it out at compile time.

Reply:
@bhatiam12 I tried that. I took the SkillRepository @Inject out and renamed the skillRepo variable to skillRepository and it isn't compiling